### ISSUE TITLE
An annoying error while executing a sync function (neovim)

### ISSUE THREAD

**Issue by andvarfolomeev** [untitled-ai/jupyter_ascending.vim#7]
When I save the file, the ```jupyter_ascending#sync()``` function is is executed with an error . To be fair, the files *.sync.py and *.sync.ipynb do sync.

In ```:message```:
```Error detected while processing function <SNR>88_on_stdout:                                           line    2:                                                                                            E730: using List as a String                                                                          [JupyterAscending]                                                                                    "1_4_2.sync.py" 22L, 367C written ```

In ```$ nvim --version```:
```
NVIM v0.4.3
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -g -O2 -fdebug-prefix-map=/build/neovim-gOb7vg/neovim-0.4.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=1 -DDISABLE_LOG -Wdate-time -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim-gOb7vg/neovim-0.4.3/build/config -I/build/neovim-gOb7vg/neovim-0.4.3/src -I/usr/include -I/usr/include/lua5.1 -I/build/neovim-gOb7vg/neovim-0.4.3/build/src/nvim/auto -I/build/neovim-gOb7vg/neovim-0.4.3/build/include
Compiled by team+vim@tracker.debian.org

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

If it matters, then I use wsl 2.

How can I get rid of this error? It does not affect work, but errors are written every time and this is annoying. I hope for a quick reply, because the plugin is awesome and there are no analogues.

**Comment by joshalbrecht** [comment]
@tjdevries Any thoughts? 

I suspect this has started to happen because we now print stuff when the command runs.

I suppose we could stop printing stuff when the command runs, but the output does need to be handled correctly, as we do need to communicate to users if there is an error.

**Comment by tjdevries** [comment]
Can you try with updated nvim? 0.4.3 is quite outdated. Latest stable version is 0.5.1

I can take a look otherwise

**Comment by ada0l** [comment]
@tjdevries, Of course, I can update my neovim, but in general this is a strange solution to the problem, because this is the latest version in the repository of Ubuntu 20.04.03 LTS.

**Comment by tjdevries** [comment]
Well I was just hoping that would fix the problem 😂 otherwise I can try and get an older appimage and test it out. I couldn't reproduce on my newer nvim (at least at first)
